src/utils/query-mapper.js

```diff
@@ -0,0 +1,9 @@
+const mapQueryToMeteorRequest = ({ date, count_only, were_dangerous_meteors }) => ({
+    date: typeof date !== 'undefined' ? new Date(date) : null,
+    countOnly: typeof count_only !== 'undefined' && count_only === 'true',
+    wereDangerousMeteors: typeof were_dangerous_meteors !== 'undefined' && were_dangerous_meteors === 'true'
```

Comment:
it seems that the first condition does not make sense because if the value is not undefined, then it is not `true`
```js
countOnly:  count_only === 'true',
wereDangerousMeteors: were_dangerous_meteors === 'true'
```

Reply:
Updated.